### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: contrib/example-backend/controllers/clusterbinding/clusterbinding_reconcile.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ type reconciler struct {
 	createClusterRoleBinding func(ctx context.Context, binding *rbacv1.ClusterRoleBinding) (*rbacv1.ClusterRoleBinding, error)
 	updateClusterRoleBinding func(ctx context.Context, binding *rbacv1.ClusterRoleBinding) (*rbacv1.ClusterRoleBinding, error)
 
-	getRoleBinding    func(name, namespace string) (*rbacv1.RoleBinding, error)
+	getRoleBinding    func(ms, name string) (*rbacv1.RoleBinding, error)
```
Comment (s-urbaniak) [inline contrib/example-backend/controllers/clusterbinding/clusterbinding_reconcile.go:51]: s/ms/ns

File: contrib/example-backend/controllers/serviceexport/serviceexport_reconcile.go [hunk 2]
```diff
@@ -30,6 +35,10 @@ import (
 	"github.com/kube-bind/kube-bind/pkg/apis/third_party/conditions/util/conditions"
 )
 
+const (
+	soureceSpecHashAnnotationKey = "kube-bind.io/source-spec-hash"
```
Comment (s-urbaniak) [inline contrib/example-backend/controllers/serviceexport/serviceexport_reconcile.go:39]: sourceSpecHashAnnotationKey ?

File: contrib/example-backend/controllers/clusterbinding/clusterbinding_reconcile.go (hunk 1 above)
Comment (sttts) [reply contrib/example-backend/controllers/clusterbinding/clusterbinding_reconcile.go:51]: fixed

File: contrib/example-backend/controllers/serviceexport/serviceexport_reconcile.go (hunk 2 above)
Comment (sttts) [reply contrib/example-backend/controllers/serviceexport/serviceexport_reconcile.go:39]: fixed

**Review by s-urbaniak — APPROVED** [review]

**Pull request merged**